crates/ol/state-types/src/ledger.rs

```diff
@@ -0,0 +1,140 @@
+//! Accounts system types.
+//!
+//! This uses the "transitional" types described in the OL STF spec.
```

Comment:
"transitional" as this is temporary?

Reply:
"Transitional" as in this is what we're using before we transition to having an actual account system where we only load in part of the state at a time.